masterlist.yaml

```diff
@@ -550,12 +565,16 @@ plugins:
       - crc: 0x17CF8EB8
         util: SSEEdit v3.2.1
   - name: 'Improved closefaced helmets for Undeath SSE.esp'
-    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/14614/']
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/14614/'
+        name: 'Lexy''s LOTD Special Edition Patches on Nexus Mods'
     clean:
       - crc: 0xCC6D50A9
         util: SSEEdit v3.2.1
   - name: 'MLU.esp'
```

Comment:
This plug-in name mlu.esp is also used by Morrowloot Ultimate simplified and Another sorting mod 2017 has a replacer esp.
Should I add a link for those mods as well?

Reply:
Yes, I think that would be useful for disambiguation.